docs/src/components/landing/CardGrid.js

```diff
@@ -37,12 +37,16 @@ const cardContentRootStyle = (imageUrl) => ({
 });
 
 export default function CardGrid(props) {
-  const { content } = props;
+  const { content, dark } = props;
   return (
     <Box
-      sx={{
-        bgcolor: (theme) => (theme.palette.mode === 'dark' ? 'primaryDark.900' : 'grey.50'),
-      }}
+      sx={
+        dark
```

Comment:
I don't understand the whole context here, but having a `dark` prop and then still checking for `theme.palette.mode === 'dark'` feels a bit weird?

Reply:
The `dark` prop is to specify if the grid should have a darker background relative to the other sections on the landing page; agreed that the naming is confusing. 

I'll rename the prop to `darker`